_config.yml

```diff
@@ -104,108 +44,20 @@ defaults:
       toc_min: 2
       toc_max: 3
       tree: true
+  # Hide Enterprise content from sitemap, and include banner in stubs that
+  # content has moved to mirantis
   - scope:
       path: "datacenter"
     values:
       enterprise: true
-  # Latest Enterprise Platform Release
-  - scope:
-      path: "ee"
-    values:
-      ucp_org: "docker"
-      ucp_repo: "ucp"
-      dtr_org: "docker"
-      dtr_repo: "dtr"
-      ucp_version: "3.2.6"
-      dtr_version: "2.7.6"
-      win_latest_build: "docker-19.03.5"
-  # Previous DTR Releases
-  - scope:
-      path: "datacenter/dtr/2.6"
-    values:
-      hide_from_sitemap: true
-      dtr_org: "docker"
-      dtr_repo: "dtr"
-      dtr_version: "2.6.12"
-  - scope:
-      path: "datacenter/dtr/2.5"
-    values:
-      hide_from_sitemap: true
-      dtr_org: "docker"
-      dtr_repo: "dtr"
-      dtr_version: "2.5.16"
-  - scope:
-      path: "datacenter/dtr/2.4"
-    values:
-      hide_from_sitemap: true
-      dtr_org: "docker"
-      dtr_repo: "dtr"
-      dtr_version: "2.4.14"
-  - scope:
-      path: "datacenter/dtr/2.3"
-    values:
-      hide_from_sitemap: true
-      dtr_org: "docker"
-      dtr_repo: "dtr"
-      dtr_version: "2.3.11"
-  - scope:
-      path: "datacenter/dtr/2.2"
-    values:
-      ucp_version: "2.1"
-      dtr_version: "2.2"
-      docker_image: "docker/dtr:2.2.11"
-  - scope:
-      path: "datacenter/dtr/2.1"
-    values:
-      ucp_version: "2.0"
-      dtr_version: "2.1"
-  - scope:
-      path: "datacenter/dtr/2.0"
-    values:
-      ucp_version: "1.1"
-      dtr_version: "2.0"
-  # Previous UCP Releases
-  - scope:
-      path: "datacenter/ucp/3.1"
-    values:
-      hide_from_sitemap: true
-      ucp_org: "docker"
-      ucp_repo: "ucp"
-      ucp_version: "3.1.12"
-  - scope:
-      path: "datacenter/ucp/3.0"
-    values:
-      hide_from_sitemap: true
-      ucp_org: "docker"
-      ucp_repo: "ucp"
-      ucp_version: "3.0.16"
-  - scope:
-      path: "datacenter/ucp/2.2"
-    values:
-      hide_from_sitemap: true
-      ucp_org: "docker"
-      ucp_repo: "ucp"
-      ucp_version: "2.2.22"
-  - scope:
-      path: "datacenter/ucp/2.1"
-    values:
       hide_from_sitemap: true
-      ucp_version: "2.1"
-      dtr_version: "2.2"
-      docker_image: "docker/ucp:2.1.8"
+      title: Docker Enterprise moved to Mirantis
   - scope:
-      path: "datacenter/ucp/2.0"
-    values:
-      hide_from_sitemap: true
-      ucp_version: "2.0"
-      dtr_version: "2.1"
-      docker_image: "docker/ucp:2.0.4"
-  - scope:
-      path: "datacenter/ucp/1.1"
+      path: "ee"
     values:
+      enterprise: true
```

Comment:
Setting enterprise to `true` for these paths automatically includes the `_includes/enterprise_moved.html` include